### DISCUSSION TITLE
Update release tag (3.0.0)

### DISCUSSION THREAD

**Discussion by RichardTMiles** [chimurai/http-proxy-middleware#931] [General]
Running `npm install -D http-proxy-middleware` does not install the latest 3.0.0. I would like to see tag v. ^3 become the head.

**Comment by chimurai** [comment]
[3.0.0](https://www.npmjs.com/package/http-proxy-middleware/v/3.0.0) is released

https://www.npmjs.com/package/http-proxy-middleware?activeTab=versions
